sdks/python/README.md

```diff
@@ -257,40 +257,38 @@ Some interesting classes to navigate to:
 
 ## Some Simple Examples
 
-### Hello world
```

Comment:
Hello world is used a sub-title in other examples as well. Renaming that to basic pipeline is fine, however that should happen everywhere. You can either undo this change or updates all hello worlds accordingly in the other examples.